modules/caddytls/tls.go

```diff
@@ -457,7 +457,7 @@ func (t *TLS) moveCertificates() error {
 	}
 
 	// get list of used CAs
-	var oldCANames []string
+	oldCANames := make([]string, 0, len(oldAcmeCas))
```

Comment:
Similarly, I suppose it's OK if the capacity isn't exactly correct, right? Here, the realized length might be slightly under the capacity. But again this is not a hot path so it's probably fine :P